### PR TITLE
GCF file split: pub/sub

### PR DESCRIPTION
**Pull request by ace-n**

### CODE REVIEW

File: functions/pubsub/publish-message/pom.xml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    <maven.compiler.source>11</maven.compiler.source>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.cloud</groupId>
+        <artifactId>libraries-bom</artifactId>
+        <version>4.4.1</version>
```
Comment (kurtisvg) [inline functions/pubsub/publish-message/pom.xml:45]: nit: I think this is up to 5.0 now

File: functions/pubsub/subscribe-to-topic/src/main/java/functions/eventpojos/PubSubMessage.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+package functions.eventpojos;
+
+// [START functions_helloworld_pubsub_message]
+import java.util.Map;
+
+public class PubSubMessage {
+  // Cloud Functions uses GSON to populate this object.
+  // Field types/names are specified by Cloud Functions
+  // Changing them may break your code!
+  String data;
```
Comment (kurtisvg) [inline functions/pubsub/subscribe-to-topic/src/main/java/functions/eventpojos/PubSubMessage.java:26]: These fields should be private?

File: functions/pubsub/publish-message/pom.xml (hunk 1 above)
Comment (ace-n) [reply functions/pubsub/publish-message/pom.xml:45]: `5.1.0` - done.

File: functions/pubsub/subscribe-to-topic/src/main/java/functions/eventpojos/PubSubMessage.java (hunk 2 above)
Comment (ace-n) [reply functions/pubsub/subscribe-to-topic/src/main/java/functions/eventpojos/PubSubMessage.java:26]: Good catch - fixed.

File: functions/pubsub/publish-message/pom.xml [hunk 3]
```diff
@@ -100,6 +86,24 @@
       <version>${powermock.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.6.2</version>
+      <scope>test</scope>
+    </dependency>
```
Comment (lesv) [inline functions/pubsub/publish-message/pom.xml:94]: You really don't wish to mix JUnit4 & Junit5

File: functions/pubsub/subscribe-to-topic/pom.xml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      <version>1.0.1</version>
+    </dependency>
+
+    <!-- The following dependencies are only required for testing -->
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.6.2</version>
+      <scope>test</scope>
+    </dependency>
```
Comment (lesv) [inline functions/pubsub/subscribe-to-topic/pom.xml:53]: Don't mix JUnit4 & 5

File: functions/pubsub/publish-message/pom.xml (hunk 3 above)
Comment (ace-n) [reply functions/pubsub/publish-message/pom.xml]: Done.

File: functions/pubsub/subscribe-to-topic/pom.xml (hunk 4 above)
Comment (ace-n) [reply functions/pubsub/subscribe-to-topic/pom.xml]: Done.

**Pull request merged**